multiversx_sdk/core/transactions_factories/relayed_transactions_factory.py

```diff
@@ -72,6 +72,35 @@ def create_relayed_v2_transaction(self,
             options=inner_transaction.options
         )
 
+    def create_relayed_v3_transaction(self,
+                                      relayer_address: IAddress,
+                                      inner_transactions: Sequence[ITransaction]) -> Transaction:
+        if len(inner_transactions) == 0:
+            raise InvalidInnerTransactionError("The are no inner transactions")
+
+        inner_txs_gas_limit = 0
+        for inner_transaction in inner_transactions:
+            if not inner_transaction.signature:
+                raise InvalidInnerTransactionError("The inner transaction is not signed")
+
+            if inner_transaction.relayer != relayer_address.to_bech32():
+                raise InvalidInnerTransactionError("The inner transaction has an incorrect relayer address")
+
+            inner_txs_gas_limit += inner_transaction.gas_limit
+
+        move_balances_gas = self._config.min_gas_limit * len(inner_transactions)
+        gas_limit = self._config.min_gas_limit + move_balances_gas + inner_txs_gas_limit
+
+        return Transaction(
+            sender=relayer_address.to_bech32(),
+            receiver=relayer_address.to_bech32(),
+            value=0,
+            gas_limit=gas_limit,
+            chain_id=self._config.chain_id,
+            inner_transactions=inner_transactions,
+        )
```

Comment:
No other `options` or something like that, correct?

Reply:
that should be right.